programs/stake/src/stake_state.rs

```diff
@@ -1033,7 +1033,8 @@ fn validate_delegated_amount(
     // With the `stake_allow_zero_undelegated_amount` feature, stake accounts
     // may be initialized with a lower balance, so check the minimum in this
     // function, on delegation.
-    if feature_set.is_active(&stake_allow_zero_undelegated_amount::id())
+    if (feature_set.is_active(&stake_allow_zero_undelegated_amount::id())
+        || feature_set.is_active(&feature_set::stake_raise_minimum_delegation_to_1_sol::id()))
         && stake_amount < crate::get_minimum_delegation(feature_set)
     {
         return Err(StakeError::InsufficientStake.into());
```

Comment:
I could've *not* added `|| feature_set.is_active(&feature_set::stake_raise_minimum_delegation_to_1_sol::id()` to this `if`, but then I would *require* that the `stake_allow_zero_undelegated_amount` feature got activated first.

Granted, it likely will get activated first, but I didn't want to require it here.

Reply:
Sounds good to me